Styled font weight and response box

Solance opened the pull request:
Changed the response box style to be the same as in the English version.
Changed the fonts weight - numbers and English letters are closer to Hebrew letters.

ishefi commented on static/css/style.css:
```diff
@@ -48,6 +48,16 @@ body {
+
+#response p {
+    font-style: italic;
+}
```
this is what it looks like before winning (the line below the guess box) - can we remove that?

<img width="908" alt="image" src="https://user-images.githubusercontent.com/11841271/159566668-e130c4be-6fb2-40a2-9357-ee23c0fd70d3.png">

Solance replied:
Right, that's something that is probably not possible to change with only CSS and I unfortunately don't do Python :) Can drop the border.

ishefi commented on static/css/style.css:
```diff
@@ -48,6 +48,16 @@ body {
+    border: 1px solid #716f6f;
+    padding-right: 20px;
+    padding-left: 5px;
+}
```
(any ideas on how to make the design cleaner?)

Solance replied:
On my machine (different browsers) it looks different, only italic. 

<img width="895" alt="Screenshot 2022-03-22 at 22 40 52" src="https://user-images.githubusercontent.com/20438330/159581431-bc977b92-c971-4280-8229-e1593f065807.png">

The color is the same one as in the English version, but I will try to think of a better one, as well as some cleaner design (I'm a bit limited here but will try!).

ishefi replied:
Now there's no background in light-mode
<img width="840" alt="image" src="https://user-images.githubusercontent.com/11841271/160285155-866767b7-3488-4211-abf9-5ccd2de6c4bb.png">

ishefi commented on static/css/style.css:
```diff
@@ -4,7 +4,7 @@ body, input {
     font-family:-apple-system-font, BlinkMacSystemFont, 'Helvetica Neue', sans-serif;
     text-rendering: optimizeLegibility;
-    font-weight:200;
+    font-weight:300;
```
this affects mostly numbers - I actually liked them being thin, but I know nothing about design. So, you think it's better this way?

Solance replied:
Yes, I think it is much better, it was way too thin comparing to the rest of the text.

ishefi commented on static/css/style.css:
```diff
@@ -2,9 +2,9 @@ body, input {
     line-height: 27px;
     -webkit-font-smoothing: subpixel-antialiased;
-    font-family:-apple-system-font, BlinkMacSystemFont, 'Helvetica Neue', sans-serif;
+    font-family:-apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;
```
I actually like the current font better (at least on mac) - current:
<img width="383" alt="image" src="https://user-images.githubusercontent.com/11841271/160285191-b58ec332-e96f-4282-941b-f495fd25a0b6.png">

new:
<img width="384" alt="image" src="https://user-images.githubusercontent.com/11841271/160285183-7358d09f-968a-41a3-a2ca-c91aa3b16a64.png">

Also, line spacing in the paragraph is a bit too large + font size is too small

ishefi replied:
I liked the colors and the design of the "guess" button + textbox!

Solance commented:
I fixed the response box background in light mode. I actually considered removing it, that's why it was missing. But I added it again, with a slight different color. I'm afraid I can't do more than that given the limitations I have.

Solance commented:
Is there anything more I should do/change then? :)
